examples/03-ui-components/12-custom-ui/README.md

```diff
@@ -1,6 +1,6 @@
-# Custom UI
+# Using Material UI
```

Comment:
Maybe a mix? "Custom all UI elements (Material)"?

It feels like the purpose of the demo is to be a "cookbook" / showcase for how to customize all of the UI, the purpose is not necessarily "how to use Material UI". If it were the latter, we should just create a Material UI package imo. Let's try to make the title + description reflect this purpose

Reply:
So should it be a mix of MUI + the default Mantine UI elements? Or do you suggest implementing the remaining elements (link toolbar, table handles, file panel) in MUI? Since right now those are disabled.

Reply:
No sorry, I meant, maybe the title should be a mix? (you changed it to "Material UI", but I don't think that conveys the purpose of the example)

Reply:
Oh right, yeah I think "UI with Third-Party Components" is probably the most accurate for the use case that the demo is targeted at